aqua/iTerm2/Portfile

```diff
@@ -26,6 +27,9 @@ patchfiles          patch-Makefile.diff
 
 post-patch {
     reinplace "s|CODE_SIGN_IDENTITY = \".*\";|CODE_SIGN_IDENTITY = \"\";|g" ${worksrcpath}/iTerm2.xcodeproj/project.pbxproj
+
+    # Fix version number
+    reinplace "s|^3\.1\.0$|${version}|" ${worksrcpath}/version.txt
```

Comment:
When this file is updated we'll have to modify the regex too. Overwriting it seems to be a better solution.

Reply:
I assumed this was just a mishap for 3.1.2, but the version in the official binary release is correct. I do not know upstream's release process, but looking through some other tags I realized that version.txt also was not correct in the past.

Usually I like to avoid overwriting, as we would likely miss a change in the format in the future. Although in this case, overwriting to correct the version number seems justified.

Reply:
https://github.com/gnachman/iTerm2/blob/master/tools/release_stable.sh#L83-L101

Reply:
Upstream seems to be overwriting that file too. Also in 3.1.3 it's content changed to `3.1.%(extra)s` and the version becomes `Build 3.1.20171012` in the about dialog.

Reply:
They normally use `3.1.%(extra)s` and the build expands that based on the current date. The `release_stable.sh` writes a fixed version number to the file, but the change is discarded after the build and not committed.

I came to the conclusion that we really should overwrite the file to always have the expected version number.